job-dsls/jobs/kie_dailyBuild_pipeline.groovy

```diff
@@ -276,15 +280,17 @@ rm -rf \\`find \\$baseDir -type f -name "*zip"\\`
 rm -rf \\`find \\$baseDir -type d -name "gwt-unitCache"\\`
 EOT
 
-# (1) do a full build, but deploy only into local dir
+# do a full build, but deploy only into local dir
 # we will deploy into remote staging repo only once the whole build passed (to save time and bandwith)
 ./droolsjbpm-build-bootstrap/script/mvn-all.sh -B -e clean deploy -T2 -Dfull -Drelease -DaltDeploymentRepository=local::default::file://$deployDir -s $SETTINGS_XML_FILE\\
  -Dkie.maven.settings.custom=$SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true\\
  -Dgwt.memory.settings="-Xms1g -Xmx10g -Xms1g -Xss1M" --clean-up-script="$WORKSPACE/clean-up.sh"
 
-# (2) upload the content to remote staging repo
-mvn -B -e org.sonatype.plugins:nexus-staging-maven-plugin:1.6.8:deploy-staged-repository -DnexusUrl=https://repository.jboss.org/nexus -DserverId=jboss-releases-repository\\
- -DrepositoryDirectory=$deployDir -s $SETTINGS_XML_FILE -DstagingProfileId=15c3321d12936e -DstagingDescription="kie $kieVersion" -DkeepStagingRepositoryOnCloseRuleFailure=true -DstagingProgressTimeoutMinutes=80
+# unpack zip to QA Nexus
+cd $deployDir
+zip -r kiegroup .
+curl --upload-file kiegroup.zip -u $kieUnpack -v http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/service/local/repositories/kieAllBuild/content-compressed
```

Comment:
Afaik, the Jenkins URL redirects to https always - so you need to keep an eye on SSL certificate expiration even though you are not using https explicitely. Just to let you know..

Reply:
Sorry, my mistake - I confused our Nexus instance with our Jenkins master, please disregard my previous comment.